fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -427,6 +260,167 @@ def _threshold_optimization_equalized_odds(sensitive_features, labels, scores, g
     return predicted_EO_by_sensitive_feature
 
 
+_SUPPORTED_CONSTRAINTS = {
```

Comment:
This slightly confused me in `fit` where it's used because the keys here are the names of the constraints, but the values are the actual mitigation methods. Maybe `_SUPPORTED_CONSTRAINT_MITIGATIONS` ? Or something like that?

Reply:
Agreed that it might be good to change, but to something less verbose. Maybe `_SUPPORTED_MITIGATIONS` or `_SUPPORTED_APPROACHES`

Reply:
I like "supported mitigations"